MultiLineString/MultiLineString/MultiLineStringCodeFixProvider.cs

```diff
@@ -31,7 +31,6 @@ public sealed override FixAllProvider GetFixAllProvider()
 
         public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
```

Comment:
Shouldn't be async as there is no async code here. use `return Task.CompletedTask;`